### ISSUE TITLE
Parcel instead of Webpack?

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#335]
### Problem

Our build system could use an update, and formerly we were considering switching to Webpack (#170), but recently a very interesting newcomer called [Parcel](https://parceljs.org/) has come on the block.

### Solution

Research switching to Parcel instead of Webpack, and consider also whether or not there's any merit to [this comment about Rollup.js](https://news.ycombinator.com/item?id=15857063).

**Comment by sandrina-p** [comment]
I've tried Parcel earlier this year and it's cool but I think it has lot to grow compared to Webpack (now with 4th major release) that is already very very robust and well documented with a lot of integrations (plugins/loaders).

**Comment by sdegutis** [comment]
I've had a lot of success with manual Webpack configurations in the past year, and generally recommend it. I've also ran into many edge-cases with Rollup, and now only use it when Webpack won't do. (I think I passed on Parcel or ran into issues with it, but can't remember why or what.)

Speaking of which, if we upgrade to a newer Rollup version in the interim (or switch to Webpack) we already [have an opportunity](https://github.com/okTurtles/group-income-simple/pull/951#discussion_r486602094) to use new syntax like optional chaining.
